cpp/ql/lib/semmle/code/cpp/security/FunctionWithWrappers.qll

```diff
@@ -37,7 +37,7 @@ private predicate wrapperFunctionStep(
   not target.isVirtual() and
   not source.isVirtual() and
   source.hasDefinition() and
-  exists(Call call, Expr arg, Parameter sourceParam |
+  exists(FunctionCall call, Expr arg, Parameter sourceParam |
     // there is a 'call' to 'target' with argument 'arg' at index 'targetParamIndex'
     target = resolveCall(call) and
```

Comment:
Do we still need `resolveCall` here, or can we just do `getTarget` now?

Reply:
We can probably do `getTarget`, yeah.